common/ast/tests/it/testdata/expr.txt

```diff
@@ -0,0 +1,522 @@
+---------- Input ----------
+a
+---------- Output ---------
+a
+---------- AST ------------
+ColumnRef {
+    database: None,
+    table: None,
+    column: Identifier {
+        name: "a",
+        quote: None,
+    },
+}
```

Comment:
IMO, it's better to take normalized AST(i.e. the `Debug` string of AST) as output, since structure of AST may change in the future.

Reply:
It is the debug string of AST.

Reply:
Oh, my fault. I mean the `Display` string, which is normalized SQL string of an AST.